src/modules/logs/logs.effects.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { Effect, Actions, ofType } from '@ngrx/effects';
+
+import { switchMap, map, catchError, mapTo, withLatestFrom } from 'rxjs/operators';
+import { of } from 'rxjs/observable/of';
+import { interval } from 'rxjs/observable/interval';
+import { Store, select } from '@ngrx/store';
```

Comment:
Could this import be moved to the top group of imports.

Reply:
sure